### PR TITLE
Move collector into a sub-directory and add http path annotations.

### PR DESCRIPTION
**Pull request by MatthewDolan**

### CODE REVIEW

**Comment by MatthewDolan** [comment]
I don't know why circle is trying to run here, there isn't anything to build. Should we disable it?

**Review by danielamiao — APPROVED** [review]
Yes I think it's OK to bypass or disable CI in this case (I don't have admin rights/permissions to disable CI)

File: lightstep_carrier.proto [hunk 1]
```diff
@@ -2,6 +2,8 @@ syntax = "proto3";
 
 package lightstep;
 
+option go_package = "lightsteppb";
```
Comment (joeblubaugh) [inline lightstep_carrier.proto:5]: Is this the package that was already being used in Go?

Comment (MatthewDolan) [reply lightstep_carrier.proto:5]: yup!

**Pull request merged**